Plugins/Public/base_plugin/Main.cpp

```diff
@@ -2909,8 +2914,21 @@ bool ExecuteCommandString_Callback(CCmds* cmd, const wstring &args)
 				ConPrint(L" - %ls\n", base.second->basename.c_str());
 			}
 		}
+		return true;
 	}
+	else if (args.find(L"baselogin") == 0)
```

Comment:
I'm sure this was included in a different PR

Reply:
Possibly, if it does we'll resolve the merge conflict.